### DISCUSSION TITLE
Project setup with Firebase: "Verify your Firebase Project" failing no matter what I try.

### DISCUSSION THREAD

**Discussion by lostprojects** [AntlerVC/firetable#1365] [Support / Q&A]
Just starting with Rowy and trying to connect to my existing firebase project. It's consistently failing with the following error:

"Looks like you still don't have a Firebase project setup. Please click the "Open Firebase console" button above and create a project on Firebase and come back here to click the "Verify" button after."

I'm certain the two accounts are the same and have tried disconnecting and reconnecting, logging in and out, setting up new projects (both in Firebase and Rowy) and the above steps, all to no avail. 

I've been through all documentation I can find related to this and have even tried multiple browsers on the chance this is a front end issue. Still nothing.

I mentioned this on another thread by another user with the same issue yesterday. He had solved it with a simple logout and back in though. Really hoping to solve this quickly so I can get stuck in so would appreciate any advice. Thanks!

**Comment by lostprojects** [comment]
Does anybody have any ideas here? I'm still stuck on this. Rowy looks great but sadly I'm fast getting forced towards finding another solution.

**Reply by lostprojects** [reply]
@BeeBombshell are you able to offer any assistance with this please?

**Comment by zajcomm** [comment]
Same here- its soooo fraustrating, make me think to switch to supabase or ampilify!!

**Comment by lostprojects** [comment]
Yeah, this isn't cool at all, I've wasted almost a week trying to get this working or get a response. This is the only place Rowy sends users who need support.

**Comment by zajcomm** [comment]
Check out the discord channel. I faced another  issue related to secret keys and found answer there.

**Comment by harinij** [comment]
Hi @lostprojects @zajcomm - It would be super helpful if you can share a bit more information or screen recording via a tool like https://jam.dev/ so we can understand the issue further. Especially are you trying to selfhost or use via rowy.app. It's hard for us to debug the issue with the information shared above. You can also join our discord as well here: rowy.io/discord

**Comment by pasavoie11** [comment]
Same issue here

**Reply by Chadha93** [reply]
Hi @pasavoie11, could you please share a https://jam.dev/ recording here or share it on Discord by opening a new discussion in #rowy-support. Lets ensure you are all setup.
